test/testhelpers/helpers.go

```diff
@@ -39,6 +51,7 @@ func NewReport(name, namespace, testQueryName string, reportStart, reportEnd *ti
 		},
 		Spec: metering.ReportSpec{
 			QueryName:      testQueryName,
+			Inputs: inputs,
```

Comment:
You may need to setup automatic formatting on save in your IDE. I think the `make verify` check will complain about this as it's going to run `gofmt -w ...` and the `inputs` variable will be re-indented with the rest of the variables.

Reply:
I just need to push the format buttons more :-) thanks!